musicbot/bot.py

```diff
@@ -2088,7 +2088,7 @@ def check(reaction, user):
                 player.skip()  # TODO: check autopause stuff here
                 await self._manual_delete_check(message)
                 return Response(self.str.get('cmd-skip-force', 'Force skipped `{}`.').format(current_entry.title), reply=True, delete_after=30)
-            else:
+            if(param.lower() in ['force', 'f']):
```

Comment:
This logic is messy and should be rewritten rather than repeating code.

![](https://i.imgur.com/BQ6kzPh.png)

Reply:
Does the rewrite acceptable?